apps/arena/lib/arena/game/effect.ex

```diff
@@ -291,4 +305,15 @@ defmodule Arena.Game.Effect do
   end
 
   defp maybe_send_to_killfeed(entity, _pool_owner_id), do: entity
+
+  defp refresh_effect_expires_at(effect, start_time) do
+    new_expires_at =
+      if effect.expires_at do
+        start_time + effect.duration_ms
+      else
+        nil
+      end
+
+    Map.put(effect, :expires_at, new_expires_at)
+  end
```

Comment:
This function is called inside an if. Why can't you just use `and`?

Reply:
Completely agreed